ogr/services/pagure/group.py

```diff
@@ -0,0 +1,24 @@
+# Copyright Contributors to the Packit project.
+# SPDX-License-Identifier: MIT
+
+import logging
+
+from ogr.services import pagure as ogr_pagure
+
+logger = logging.getLogger(__name__)
+
+
+class PagureGroup:
+    service: "ogr_pagure.PagureService"
+
+    def __init__(self, name: str, raw_group) -> None:
+        self.name = name
+        # see https://pagure.io/api/0/#groups-tab
+        self._raw_group = raw_group
```

Comment:
I would rather call this `group_data` or something like that, I find that more self-explanatory.

Reply:
we use this terminology in other classes already, that's why I went with this